demo/ui/state/host_agent_service.py

```diff
@@ -272,7 +272,7 @@ def extract_content(
         elif p.kind == 'data':
             try:
                 jsonData = json.dumps(p.data)
-                if 'kind' in p.data and p.data['kind'] == 'form':
+                if 'type' in p.data and p.data['type'] == 'form':
```

Comment:
Shouldn't this be `kind` now? Or is this something else?

Reply:
Talked with Phil offline. This is part of the JSON payload in the Data part, so `type` is correct.